liblepton/src/sch2pcb.c

```diff
@@ -1169,6 +1174,39 @@ pkg_to_element (FILE * f, gchar * pkg_line)
   return el;
 }
 
+
+FILE*
+sch2pcb_open_file_to_write (char *filename)
+{
+  FILE *f_out;
+
+  if ((f_out = fopen (filename, "wb")) == NULL)
+  {
+    fclose (f_out);
```

Comment:
`fclose( NULL )` will fail. Om my system it even causes `SIGSEGV`.

Reply:
Thank you, indeed, I read somewhere that the function works differently on different systems, and may even segfault on NULL arguments.